src/ui/public/visualize/visualize_legend.js

```diff
@@ -66,6 +66,18 @@ uiModules.get('kibana')
         return filters.length;
       };
 
+      $scope.showLabelCount = legendData => {
```

Comment:
Consider renaming this to `showLegendCount` for consistency with `vis.params.addLegendCount`.